pom.xml

```diff
@@ -56,7 +56,7 @@
         <asciidoctorj-pdf.version>1.5.0-alpha.11</asciidoctorj-pdf.version>
 
         <!-- Ignite -->
-        <igniteVersion>2.1.0</igniteVersion>
+        <igniteVersion>2.3.0</igniteVersion>
```

Comment:
Is this needed for this PR? You should keep all the changes in logical commits

Reply:
You right. Should I revert this change?

Reply:
Do you use 2.3 version on your project? You have to revert the change if you not need 2.3 version.

Reply:
> You right. Should I revert this change?
 
Yes, please. Revert all the things that are not needed for the Multiget or, at least, extract them in separate commits so that I can check and apply and them separately.

Reply:
Done